app/models/attendance.rb

```diff
@@ -5,13 +5,33 @@ class Attendance < ActiveRecord::Base
 
   # Scopes #####################################################################
 
+  scope :pending_excuses, -> { where(excused: nil).
```

Comment:
Style/MultilineBlockLayout: Block body expression is on the same line as the block start.